apis/v1alpha1/httproute_types.go

```diff
@@ -331,7 +350,7 @@ type HTTPRouteFilter struct {
 	// group indicates that the resource is "configmaps".
 	// ExtensionRef MUST NOT be used for core and extended filters.
 	// +optional
-	ExtensionRef *RouteFilterExtensionObjectReference `json:"extensionRef"`
+	ExtensionRef *LocalObjectReference `json:"extensionRef"`
 
 	// Filter-specific configuration definitions for core and extended filters
 
```

Comment:
`RequestHeader` and `RequestMirror` appear to be optional (i.e. pointers). If so, set +optional and omitempty for each.

Reply:
Updated, thanks!